pkgs/by-name/aq/aquamarine/package.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  hwdata,
+  hyprutils,
+  hyprwayland-scanner,
+  libdisplay-info,
+  libdrm,
+  libffi,
+  libGL,
+  libinput,
+  libseat,
+  mesa,
+  nix-update-script,
+  pixman,
+  pkg-config,
+  udev,
+  wayland,
+  wayland-protocols,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "aquamarine";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "hyprwm";
+    repo = "aquamarine";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rux5XA+ixI0fuiQGSOerLKxsW2D8cfjmP1B7FY24xF8=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    hyprwayland-scanner
+    pkg-config
+  ];
+
+  buildInputs = [
+    hyprutils
+    libdisplay-info
+    libdrm
+    libffi
+    libGL
+    libinput
+    libseat
+    mesa
+    pixman
+    udev
+    wayland
+    wayland-protocols
+  ];
+
+  depsBuildBuild = [ hwdata ];
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  cmakeBuildType = "RelWithDebInfo";
+
+  dontStrip = true;
+  doCheck = false;
```

Comment:
Iirc it is false by default. The reason is the tests are faulty/no longer relevant. We can either remove this or add a comment explaining why it is explicitly set.